config/dev.secret.exs.example

```diff
@@ -0,0 +1,6 @@
+use Mix.Config
+
+config :remote_retro, :google_oauth,
+  client_id: "<Client Id>",
+  client_secret: "<Client Secret>",
+  redirect_uri: "http://localhost:4000/auth/google/callback"
```

Comment:
also, @nicholalexander! there are a few undesirable line-endings in this PR; might want to configure your editor such that those don't appear!